src/drivers/serac.cpp

```diff
@@ -15,113 +15,97 @@
 #include <fstream>
 #include <iostream>
 #include <memory>
+#include <string>
 
-#include "CLI11/CLI11.hpp"
+#include "axom/core.hpp"
 #include "coefficients/loading_functions.hpp"
 #include "coefficients/traction_coefficient.hpp"
+#include "infrastructure/cli.hpp"
 #include "infrastructure/initialize.hpp"
+#include "infrastructure/input.hpp"
 #include "infrastructure/logger.hpp"
 #include "infrastructure/terminator.hpp"
 #include "mfem.hpp"
 #include "numerics/mesh_utils.hpp"
 #include "physics/nonlinear_solid.hpp"
+#include "physics/utilities/equation_solver.hpp"
 #include "serac_config.hpp"
 
-int main(int argc, char* argv[])
-{
-  auto [num_procs, rank] = serac::initialize(argc, argv);
+namespace serac {
 
+//------- Input file -------
+
+void defineInputFileSchema(std::shared_ptr<axom::inlet::Inlet> inlet, int rank)
```

Comment:
If I understand, this is where a driver code would add any input toggles that are not in the physics. It is probably a good idea to move the mesh stuff to another helper function in `numerics/mesh_utils.cpp`